packages/core/story-structure.js

```diff
@@ -53,29 +53,34 @@ const s = [
             ],
           },
           { n: 'DataSpreadsheet', s: ['c/DataSpreadsheet'] },
-          { n: 'TruncatedList', s: ['c/TruncatedList'] },
-          { n: 'DelimitedList', s: ['c/DelimitedList'] },
           { n: 'Decorator', s: ['c/Decorator'] },
+          { n: 'DelimitedList', s: ['c/DelimitedList'] },
```

Comment:
```suggestion
          { n: 'Delimited list', s: ['c/DelimitedList'] },
```

Reply:
Good catch, I found a few others ( Inlinetip and Non-linear reading )